packages/artillery/package.json

```diff
@@ -44,7 +44,7 @@
   },
   "scripts": {
     "test:unit": "tap --no-coverage --timeout=420 --color test/unit/*.test.js",
-    "test:acceptance": "export ARTILLERY_TELEMETRY_DEFAULTS='{\"source\":\"test-suite\"}' && tap --no-coverage --timeout=420 test/cli/*.test.js && bash test/lib/run.sh && tap --no-coverage --color test/testcases/plugins/*.test.js",
+    "test:acceptance": "export ARTILLERY_TELEMETRY_DEFAULTS='{\"source\":\"test-suite\"}' && tap --no-coverage --timeout=420 test/cli/*.test.js && bash test/lib/run.sh && tap --no-coverage --color test/testcases/plugins/*.test.js && tap --no-coverage --color --timeout=3000 test/publish-metrics/*/*.test.js",
```

Comment:
do the timeouts need to be this long? (here and in https://github.com/artilleryio/artillery/pull/2718/files#diff-be85fb016971de4211dd6960474754c38e876ae617df1435e47e3b26261e2847R9)